<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coordinador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="baseSize">
    <size>
     <width>3</width>
     <height>1</height>
    </size>
   </property>
   <widget class="QLabel" name="lbHola">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;b&gt;Hola Sr.&lt;b&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>608</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>37</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QPushButton" name="btCursosAsistentes">
         <property name="text">
          <string>Cursos con 
más asistentes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btLTDpto">
         <property name="text">
          <string>LTs por
departamento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btLTReprobaron">
         <property name="text">
          <string>LTs que
reprobaron</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btLTCurso">
         <property name="text">
          <string>LTs por
curso</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btNotasLT">
         <property name="text">
          <string>Notas por
LT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btCursosAvance">
         <property name="minimumSize">
          <size>
           <width>166</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Cursos con 
menos avance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btLTAprobaron">
         <property name="text">
          <string>LTs que
aprobaron</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btHistoricoDpto">
         <property name="text">
          <string>Histórico por
departamento</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btCrear">
         <property name="text">
          <string>Crear curso</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btConsultar">
         <property name="text">
          <string>Consultar curso</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btMatricular">
         <property name="text">
          <string>Matricular LT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btANular">
         <property name="text">
          <string>Anular matricula LT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btEditar">
         <property name="text">
          <string>Editar curso</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btEliminar">
         <property name="text">
          <string>Eliminar curso</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btCertificado">
         <property name="text">
          <string>Generar Certificado
por LT</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>37</number>
     </property>
     <item>
      <widget class="QLabel" name="lbReportes">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Reportes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbReportes_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Gestión de Cursos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbReportes_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Certificados&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
